.zuul.yaml

```diff
@@ -32,6 +32,7 @@
 - job:
     name: periodic-watcher-operator-validation-master
     parent: watcher-operator-base
+    nodeset: centos-9-medium-2x-centos-9-crc-cloud-ocp-4-18-1-3xl
```

Comment:
I think you also need to modify watcher-operator-validation-master in line 225.

Reply:
added